test/e2e/requirements.txt

```diff
@@ -0,0 +1 @@
+acktest @ git+https://github.com/aws-controllers-k8s/test-infra.git@38ce32256cc2552ab54e190cc8a8618e93af9e0c
```

Comment:
We bumped botocore version in `45dfc05236f2ce63e8404c196ea63105dc1404aa`, might be worth to use

Reply:
done